docs/content/en/docs/Tutorials/share-tutorial.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Try out the share functionality in Reva"
```

Comment:
'Try out the OCM share functionality in Reva'. Please mention OCM everywhere. The user shares are different.

Reply:
Ok